### ISSUE TITLE
Exercise #2 Test case 6_1

### ISSUE THREAD

**Issue by limefreak** [nus-cs2030/2324-s1#28]
My output in codecrunch vs my output in my jshell. Weird how I was able to pass through 6 levels without the classes.
What am I missing? 

![image](https://github.com/nus-cs2030/2324-s1/assets/97215727/7e6d7819-4a14-4d0e-b7c4-b38eaa196859)

![image](https://github.com/nus-cs2030/2324-s1/assets/97215727/959d2709-2514-4cd6-ba2a-512102c77efb)

**Comment by AbdulrahmanAlRammah** [comment]
CodeCrunch is checking how your mass in SolidSphere and SolidCuboid is being calculated and your current method is not what is desired. You have to create the SolidImpl class.

**Comment by limefreak** [comment]
I did create SolidImpl class, and have a mass method in there that is referenced by both SolidSphere and SolidCuboid.

**Comment by nearsea00** [comment]
This might be silly, but can you check if you included the SolidImpl.java file in your submission? I just tried opening all my files in Jshell except for the SolidImpl class and it returns the exact same error as the screenshot you posted.

**Comment by limefreak** [comment]
That's something interesting to look into... hmmmmm 
![image](https://github.com/nus-cs2030/2324-s1/assets/97215727/1bcd8e9e-aec3-4621-a607-5b771a2da462)

**Comment by limefreak** [comment]
Resolved! Thanks for all the help!

Upon nearsea00's reply, I had the feeling that a new SolidImpl.java file is used to replace what we implemented. This means that 
1. My SolidImpl.java is not the same as their SolidImpl.java
2. As consequence of 1, SolidCuboid.java and SolidSphere.java couldn't use SolidImpl's mass() method.
3. As consequence of 2, both files fail to compile and can't be found.

How I fixed:
Change all methods and property types from "Double" to "double". 
For example: 
Double mass() {
...
}

 to 

double mass() {
...
}

Will close thread now. Thanks again to anyone that read this.
